packages/workflow/src/TelemetryHelpers.ts

```diff
@@ -200,6 +203,24 @@ export function generateNodesGraph(
 			position: node.position,
 		};
 
+		if (runData?.[node.name]) {
+			const runs = runData[node.name] ?? [];
+			nodeItem.runs = runs.length;
+
+			nodeItem.items_total = runs.reduce((total, run) => {
```

Comment:
We have 365 lines of code in this function. Should we move this part to a helper so we do not make matters worst? Maybe something like:

`
	nodeItem.items_total = getNumberOfItemsInRun(runs)
`